pom.xml

```diff
@@ -548,7 +548,7 @@
             <dependency>
                 <groupId>com.facebook.presto.hadoop</groupId>
                 <artifactId>hadoop-apache2</artifactId>
-                <version>2.7.4-7</version>
+                <version>2.7.4-8-beinan</version>
```

Comment:
oh, this PR is a dependency of:
https://github.com/prestodb/presto-hadoop-apache2/pull/43

Reply:
We have updated the version to 2.7.4-8 in another pr